package.json

```diff
@@ -18,7 +18,8 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "react-bootstrap": "^0.30.8"
   },
```

Comment:
If we're trying to shield users from having to use react-bootstrap in their application, then I think we should not require them to explicitly install react-bootstrap as a dependency of their project by making it peer-dependency